pallets/proofs-dealer/src/benchmark_proofs.rs

```diff
@@ -30,6 +32,11 @@ pub fn get_root() -> Vec<u8> {
     hex::decode("912896cfb8a6b305bd4865e5412b31b102b79b5f9652df471a9c93477e54d916").expect("Root should be a decodable hex string")
 }
 
+#[rustfmt::skip]
+pub fn get_user_account() -> AccountId32 {
+	<AccountId32 as Ss58Codec>::from_ss58check("5CombC1j5ZmdNMEpWYpeEWcKPPYcKsC1WgMPgzGLU72SLa4o").expect("User account should be a decodable string")
+}
```

Comment:
This change should actually go in the `benchmark_proofs_template.rs` file, and be updated by the script.